alf/ext/act_backward.py

```diff
@@ -26,7 +26,10 @@
                     verbose=True)
 
         relu_backward_cuda = _ext.relu_backward
-    except ImportError:
+    except (ImportError, OSError):
```

Comment:
perhaps we can print the exception in a warning so that people knows what to do if fused_linear_act is needed.